data/hai/hai reveal 4 middle.txt

```diff
@@ -1391,7 +1397,37 @@ mission "Hai Reveal [A10] Demanding Satisfaction"
 			choice
 				`	(Walk to her.)`
 			`	"Captain <first> <last> led great fleets during a civil war of the humans. We will have the honor of fighting alongside this warrior of the humans."`
-			`	He looks at you appraisingly. "That is well, but there is a matter yet to be addressed. We cannot just 'waltz' through Hai and human space without being blasted. How will you convince your Elders to allow us passage?"`
+			branch nobody
+				not "Wanderers: Defend Vara Ke'sok: done"
+			`	"Indeed, we have had the honor of fighting against this warrior of humans. Quite a troublemaker on the battlefield." He looks at you. "Your professor friend was chatty before you left, and it's been easy to track you since then. We've been studying your actions for quite a while. Why are you here now, as an ally of this True Hai?"`
+			choice
+				`	"You and I have crossed swords before, but this is a matter that concerns us both: the safety and honor of the Hai."`
+					goto honor
+				`	"I fight for money and prestige. The Hai are paying plenty of both for protection against a pirate threat."`
+					goto mercenary
+				`	"I assumed furballs like you were too brain-dead to count to two."`
```

Comment:
What leads here?
Also, this seems like an interesting sentence because it uses all the homophones of to.

Reply:
The player wanting to be an ass hole leads to there. I used a sentence with a nursery rhyme feel so it is even more demeaning.

Reply:
But there’s no `label` above it. So nothing ever leads there.

Reply:
That whole `choice` section is reached if the `branch` above it is false. I tested every path through the conversation, so I do know it works.

Reply:
Sorry, I had forgotten it was a `choice`.

Reply:
This can be resolved.